build.sbt

```diff
@@ -21,14 +21,15 @@ lazy val `hello-api` = (project in file("hello-api"))
   )
 
 lazy val `hello-impl` = (project in file("hello-impl"))
-  .enablePlugins(LagomScala, SbtReactiveAppPlugin)
+  .enablePlugins(LagomScala)
   .settings(headerSettings)
   .settings(
-    dockerAliases in Docker += DockerAlias(None, None, "hello-impl", None),
+    dockerAliases in Docker += DockerAlias(None, None, "hello-lagom", None),
```

Comment:
Let's rethink how we name the images so we can easily identify them (also the java one) when browsing the docker registry.